### ISSUE TITLE
owasp-crs-v030301-id932130-rce triggers with () on a form's input field

### ISSUE THREAD

**Issue by cccnc** [coreruleset/coreruleset#3103]
Hi, 

I'm using GLPI (https://glpi-project.org/) and when I'm about to create a new ticket case we've seen that whenever we input an empty pair of parenthesis [()], (image below)

![image](https://user-images.githubusercontent.com/123103210/213505540-f442a52f-5f49-4775-9ba7-e7acf827f8cb.png)

 it triggers the following rule:

  "priority": 3,
      "outcome": "DENY",
      "preconfiguredExprIds": [
        "owasp-crs-v030301-id932130-rce"
      ],
      "name": "glpi-owasp-rules",
      "configuredAction": "DENY"
    }

It doesn't matter if there is something inside the parenthesis, it triggers the deny rule, which seems a bit odd given that there is no further pattern besides the parenthesis.

**Comment by franbuehler** [comment]
When I test with your input `()` as an argument against our [sandbox](https://coreruleset.org/docs/development/sandbox/), the request doesn't trigger any rule:

Nightly build:
`curl --header "x-crs-paranoia-level: 1" --header "x-format-output: txt-matched-rules" --header "x-crs-version: nightly"  'http://sandbox.coreruleset.org/?arg=%28%29'`

3.3.2:
`curl --header "x-crs-paranoia-level: 1" --header "x-format-output: txt-matched-rules" --header "x-crs-version: 3.3.2"  'http://sandbox.coreruleset.org/?arg=%28%29'`

Which version are you using? Do you have more information? I can not reproduce your issue.

**Comment by cccnc** [comment]
Thanks for the info, here is what I think is happening:

on the input field, there are a coulpe <p></p> tags inserted by the framework, and when we inserted () we ended up with >() which according to the rule would match something like:
>(foo)    Process substitution

and it does indeed find a match in the regex pattern for:

>()
https://regex101.com/r/YeifBP/1

I don't know if that's the intended purpose for the rule, because it's a bit vague to block traffic based on that patter on P1, right now for example we can't use parentheses in the input field to add supplemental information.

**Comment by franbuehler** [comment]
There are actually frameworks that add tags to the request, which are then blocked by the CRS.

In your case, I would add a tuning rule that removes the target `ARGS:<name-of-description-field>` from the rule 932130 with:
`SecRuleUpdateTargetById 932130 !ARGS:<name-of-description-field>` and add this tuning rule AFTER the CRS rules.

I searched for false positives reports of rule 932130 in old issues and didn't find similar cases.
So, I don't think we're going to change the CRS rule here unless there are other opinions from the team.

**Comment by dune73** [comment]
I second your assessment @franbuehler and sorry for the inconvenience @cccnc. And thank you for reporting even if we are not likely to take action. It helps us having FPs in our records for future reference or rules evolving into serial offenders.
